src/coreComponents/physicsSolvers/fluidFlow/IsothermalCompositionalMultiphaseFVMKernels.hpp

```diff
@@ -476,8 +476,8 @@ class FaceBasedAssemblyKernel : public FaceBasedAssemblyKernelBase
       : stencilSize( size ),
       numFluxElems( numElems ),
       compFlux( numComp ),
-      dCompFlux_dP( size, numComp ),
-      dCompFlux_dC( size, numComp, numComp ),
+      dCompFlux_dP( 2, numComp ),
+      dCompFlux_dC( 2, numComp, numComp ),
```

Comment:
So I can see `2` popping here and there, while previously we had `size` and `maxStencilSize`.
Could you at least create constants so we know which `2` is when we need to refactor.

Reply:
I can but this assembly pattern only works with a `two-point flux approximation` so it is important to keep in mind that it becomes a constant that can't really be changed. But I guess I will just define it directly in this file.

Reply:
Actually, what we should do (but in a separate PR) is a review of the meaning of some of the constants defined in `StencilBase` coz with all the fracture work some of them have kind of changed meaning. And we could probably remove the MPFA  stencil.

Reply:
If `size` and `maxStencilSize` have become identical, maybe add a `STENCIL_SIZE = 2`.
Eventually having a review of the constants looks nice indeed.